models/brigade.go

```diff
@@ -2,8 +2,9 @@ package models
 
 type Brigade struct {
 	// Discord config
-	ID                        int    `gorm:"AUTO_INCREMENT"`
+	ID                        int    `gorm:"primary_key;AUTO_INCREMENT"`
```

Comment:
Out of curiosity, what is the necessity of designating 'primary_key' on all the models?

Reply:
Allows for gorm to create ID as a primary key if it is creating tables(when someone first sets up the project). Looking at the docs, it may do this automatically if the field is called ID, but I'm going to test and do a separate PR for refactoring all of the gorm tags anyways.